seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java

```diff
@@ -264,8 +264,8 @@ public void testMysqlCdcCheckDataWithNoPrimaryKey(TestContainer container) {
     @TestTemplate
     @DisabledOnContainer(
             value = {},
-            type = {EngineType.SPARK, EngineType.FLINK},
-            disabledReason = "Currently SPARK and FLINK do not support multi table")
+            type = {EngineType.SPARK},
+            disabledReason = "Currently SPARK do not support multi table")
```

Comment:
spark already supports multiple tables, but not cdc

Reply:
thank you for suggestion, I've modified it.